packages/kbn-apm-synthtrace/src/scenarios/simple_logs.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { LogDocument, log } from '@kbn/apm-synthtrace-client';
+import { Scenario } from '../cli/scenario';
+
+const scenario: Scenario<LogDocument> = async () => {
+  return {
+    generate: ({ range }) => {
+      const LOG_LEVELS = ['info', 'debug', 'error'];
+      const MESSAGES = ['A simple log', 'Yet another log', 'Something went wrong'];
+
+      const services = Array(5)
+        .fill(0)
+        .map((_, idx) => log.service(`service-${idx}`));
+
+      return range
+        .interval('1m')
+        .rate(1)
+        .generator((timestamp, timestampIndex) => {
```

Comment:
It's probably better to inject the clients into the scenario. That also allows you to create scenarios with mixed data. I'm pretty sure you never want to execute a logs scenario with the APM client either.

Reply:
Really great idea, let me investigate